src/main/java/edu/umich/its/lti/TcSessionCustomData.java

```diff
@@ -0,0 +1,25 @@
+package edu.umich.its.lti;
+
+import java.util.HashMap;
+
+import javax.servlet.http.HttpServletRequest;
+
+import edu.umich.its.lti.utils.OauthCredentials;
+
+public class TcSessionCustomData extends TcSessionData{
+	
```

Comment:
If agree with Dave - if we could add an over-loaded constructor, with an additional parameter, then this is preferred to a subclass. I think a method comment should also be added to explain the additional parameter (so I'd prefer in-code comments over updates to the README here).